src/builder.js

```diff
@@ -42,6 +43,9 @@ const FormBuilder = ({
   ...props
 }) => {
   const useFormObj = useForm({ defaultValues: { formatDate: '' } })
+  const [formDataValues, setFormDataValues] = React.useState({})
+  const hasRecaptcha = form.questions.some(question => question.type === 'recaptcha')
+  const recaptchaRef = React.createRef(null)
```

Comment:
```suggestion
  const recaptchaRef = useRef(null)
```
Just dont forget to import the hook first.